oneway/one_way_anova.Rmd

```diff
@@ -1,23 +1,33 @@
 ---
 jupyter:
   jupytext:
+    formats: ipynb,Rmd
     notebook_metadata_filter: all,-language_info
     split_at_heading: true
     text_representation:
       extension: .Rmd
       format_name: rmarkdown
       format_version: '1.2'
-      jupytext_version: 1.8.2
+      jupytext_version: 1.10.1
   kernelspec:
     display_name: Python 3
     language: python
     name: python3
 ---
 
+```{python}
```

Comment:
For some reason I prefer this boilerplate lower down - maybe to separate the introduction, not-code part, from the code part.   What do you think?

Reply:
Assuming I interpreted this correctly, in the 'simulation graphs as images' commit below I've moved the 'import numpy as np (etc).' section down into the first code cell (where the raw data is read in).